examples/basic/calc.go

```diff
@@ -2,6 +2,7 @@ package calc
 
```

Comment:
Would be good to keep the calc example as is. The idea of this basic example is to have a minimal functional "first time" design.

Reply:
I reverted the changes to calc and added a new "encodings" example.  I'm not sure if these changes warrant a new example so I will get rid of it entirely if you think we don't need it.